### DISCUSSION TITLE
kie中的re如何标注，ppocrlabel中标注的没有re功能需要的id和link字段

### DISCUSSION THREAD

**Discussion by PlayerJian** [PaddlePaddle/PaddleOCR#13899] [Q&A]
kie中的训练re功能如何标注，ppocrlabel中标注的没有re功能需要的id和link字段

**Comment by UserWangZz** [comment]
目前，PPOCRLabel不支持re任务的标注，并且PaddleOCR中re任务的算法已经过时，当前较好的解决方法是使用大模型进行KIE任务解析

**Reply by Slayerxxx** [reply]
您好，请教一下pp-chatocr模型的架构。是不是最终输入给大模型的只有文字了，大模型不直接处理图像？

**Reply by UserWangZz** [reply]
是的

**Reply by Lucknicking** [reply]
请问依靠大模型来解析的话，这个出结果速度会比较慢，这个怎么解决呢
